modules/ob-about-build-strategies.adoc

```diff
@@ -0,0 +1,40 @@
+// Ths module is included in the following assembly:
+//
+// * builds/overview-openshift-builds.adoc
+
+:_content-type: CONCEPT
+[id="about-build-strategies_{context}"]
+= About build and cluster build strategies
```

Comment:
This title is probably fine since referring to both "strategies" at once feels natural in the general reference form, however if you take the object reference suggestion to change the others to "About the Build resource" and "About the BuildRun resource", it might be worth syncing this one up for consistency as well:
```suggestion
= About the BuildStrategy and ClusterBuildStrategy resources
```